README.md

```diff
@@ -33,6 +33,7 @@ UDS Core establishes a secure baseline for cloud-native systems and ships with c
 
 ### Prerequisites
 
+- A running container environment for K3D to interact with
 - [K3D](https://k3d.io/) for dev & test environments or any [CNCF Certified Kubernetes Cluster](https://www.cncf.io/training/certification/software-conformance/#logos) for production environments.
```

Comment:
Can we explicitly call out this is required for dev/test, since this prerequisites list is not specific to only dev/test? (see bullet below where EITHER k3d or a prod cluster are required)

Reply:
```suggestion
- A running container environment for K3D to interact with for dev & test environments
- [K3D](https://k3d.io/) for dev & test environments or any [CNCF Certified Kubernetes Cluster](https://www.cncf.io/training/certification/software-conformance/#logos) for production environments.
```